cards/brann_bronzebeard/brann_bronzebeard.tscn

```diff
@@ -11,12 +11,14 @@ text = "Your Battlecries trigger
 twice."
 cost = 3
 texture = ExtResource("2_1w15u")
-types = Array[int]([1])
-classes = Array[int]([0])
-rarities = Array[int]([4])
+types = Array[StringName]([&"Minion"])
+classes = Array[StringName]([&"Neutral"])
+modules = {
+"rarities": [&"Legendary"]
+}
```

Comment:
Is this good? This would be a lot better in Godot 4.3 because of typed dictionaries.